sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_analysis_client.py

```diff
@@ -25,7 +25,7 @@ class DocumentAnalysisClient(FormRecognizerClientBase):
     methods based on inputs from a URL and inputs from a stream.
 
     .. note:: DocumentAnalysisClient should be used with API versions
-        v2021-09-30-preview and up. To use API versions <=v2.1, instantiate a FormRecognizerClient.
+        2021-09-30-preview and up. To use API versions <=v2.1, instantiate a FormRecognizerClient.
```

Comment:
I think we should make this change on the rest of the clients if we make it here.

Reply:
Done

Reply:
Done